src/discof/replay/fd_replay_tile.c

```diff
@@ -656,6 +656,15 @@ restore_slot_ctx( fd_replay_tile_ctx_t * ctx,
 
   fd_solana_manifest_global_t * manifest_global
     = (fd_solana_manifest_global_t *)fd_chunk_to_laddr( fd_wksp_containing( ctx->manifest_dcache ), chunk );
+
+  /* If the bank already exists, that means that we have already
+    restored the bank for this slot. */
+  fd_bank_t * bank = fd_banks_get_bank( ctx->slot_ctx->banks, manifest_global->bank.slot );
+  if( FD_UNLIKELY( !!bank ) ) {
+    FD_LOG_NOTICE(( "The bank for slot %lu already exists, skipping slot context restore", manifest_global->bank.slot ));
+    return;
+  }
```

Comment:
hm - is the bank entirely derived from manifest? so multiple banks can get restored if a snapshot gets cancelled but manifest is already sent?